bar.hpp

```diff
@@ -0,0 +1,12 @@
+namespace test
```

Comment:
Use conditional compilation to avoid duplication of include.

Reply:
fixed